streaming/src/main/scala/org/apache/spark/streaming/dstream/DStreamCheckpointData.scala

```diff
@@ -102,8 +102,8 @@ class DStreamCheckpointData[T: ClassTag] (dstream: DStream[T])
   }
 
   /**
-   * Restore the checkpoint data. This gets called once when the DStream graph
-   * (along with its DStreams) are being restored from a graph checkpoint file.
+   * Restore the checkpoint data. This gets called once while the DStream graph
```

Comment:
I am not sure if this comment is accurate, someone from the streaming team should check it. cc @tdas